library/network/src/modules/NetworkPopup.rb

```diff
@@ -65,19 +65,23 @@ def ChooseItem(_title, items, selected)
 
       items = Builtins.maplist(items) do |i|
         device_name = NetworkInterfaces.GetValue(i, "NAME")
+
         if device_name.nil? || device_name == ""
           # TRANSLATORS: Informs that device name is not known
           device_name = _("Unknown device")
         end
+
         if Ops.greater_than(Builtins.size(device_name), 30)
           device_name = Ops.add(Builtins.substring(device_name, 0, 27), "...")
         end
+
         ip_addr = if NetworkInterfaces.GetValue(i, "BOOTPROTO") == "dhcp"
                     # TRANSLATORS: Informs that the IP address is assigned via DHCP
                     _("DHCP address")
                   else
                     NetworkInterfaces.GetValue(i, "IPADDR")
-                  end
+        end
```

Comment:
this is in fact not so nice. I expect end where it used to be...to have same indentation as if/else.